StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/DeferredIntentContext.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  DeferredIntentContext.swift
+//  StripePaymentSheet
+//
+//  Created by Nick Porter on 3/3/23.
+//
+
+import Foundation
+import StripePayments
+
+class DeferredIntentContext {
```

Comment:
Add a comment explaining why we've made this - it's just a convenience/helper to avoid passing every single one of its properties across the static methods in `PaymentSheet+API.swift`.